lib/vsc/utils/script_tools.py

```diff
@@ -129,6 +131,7 @@ def __init__(self, options, run_prologue=True, excepthook=None, **kwargs):
 
         self.nagios_reporter = None
         self.lockfile = None
+        self.MonitorClass = monitorclass
```

Comment:
can we have this all lowercase, like the other attribute names

Reply:
sure, done in https://github.com/hpcugent/vsc-utils/pull/81/commits/c7595ad59e6e1dfd9b2d76f97f7807f1d4e36847